src/panels/lovelace/cards/hui-tile-card.ts

```diff
@@ -64,8 +64,11 @@ export class HuiTileCard extends LitElement implements LovelaceCard {
       throw new Error("Specify an entity");
     }
 
+    const domain = computeDomain(config.entity);
     const supportToggle =
```

Comment:
Yes but the `toggle` tap action handler incompatible with `scene` and `button`. We should rename it too but we can't because it's a breaking change for dashboards.